src/DetailsView/details-view-container.tsx

```diff
@@ -94,9 +96,17 @@ export class DetailsViewContainer extends React.Component<DetailsViewContainerPr
 
     public render(): JSX.Element {
         if (this.shouldShowNoContentAvailable()) {
+            const headerProps: Omit<HeaderProps, 'isNarrowMode'> = { deps: this.props.deps };
             return (
                 <>
-                    <Header deps={this.props.deps} />
+                    <NarrowModeDetector
```

Comment:
It’s nice that u can use narrow mode detector like that and don’t have to duplicate a lot of code and put a lot of logic into this class.